data-prepper-plugins/saas-source-plugins/jira-source/src/main/java/org/opensearch/dataprepper/plugins/source/jira/utils/Constants.java

```diff
@@ -0,0 +1,63 @@
+package org.opensearch.dataprepper.plugins.source.jira.utils;
+
+import java.util.List;
+
+/**
+ * The type Constants.
+ */
+public class Constants {
+
+    public static final String TOKEN_LOCATION = "https://auth.atlassian.com/oauth/token";
+    public static final int TOKEN_EXPIRED = 401;
```

Comment:
Armeria, Spring, and Apache HTTP Client all have HTTP constants. Please re-use those instead of creating new ones.

Reply:
Making use of Spring HttpStatus class now